ecosystem/ffmpeg_plugin/mtl_st20p_rx.c

```diff
@@ -168,6 +168,13 @@ static int mtl_st20p_read_header(AVFormatContext* ctx) {
     return AVERROR(EIO);
   }
 
+  ret = mtl_start(s->dev_handle);
+  if (ret) {
+    err(ctx, "%s, mtl start fail\n", __func__);
```

Comment:
nit: can you print the err `ret` value in case it failed really.

Reply:
Fixed